assets/components/src/color-picker/style.scss

```diff
@@ -0,0 +1,164 @@
+/**
+ * Color Picker
+ */
+
+@import '../../../shared/scss/colors';
+@import '~@wordpress/base-styles/colors';
+
+.newspack-color-picker {
```

Comment:
I know the Components Demo is a bit of a strange case, but the `ColorPicker` examples look a bit odd filling the full width. Do you think we should put a `max-width` on the component? Or do you prefer to leave that up to the container to determine?

Reply:
I'd prefer not to add a max-width to the actual component but instead leave that up to the layout/container when it's being used.